common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1204,7 +1225,7 @@
       },
       "MISMATCH_INPUT" : {
         "message" : [
-          "The input <inputType> '<input>' does not match the format."
+          "The input <inputType> <input> does not match the format."
```

Comment:
This is the issue left by [SPARK-44299](https://github.com/apache/spark/pull/41858)
https://github.com/apache/spark/pull/41858#discussion_r1254022894

Reply:
Thank you.